api/lambda/outputs/models/api.go

```diff
@@ -229,7 +229,8 @@ type JiraConfig struct {
 
 // OpsgenieConfig defines options for each Opsgenie output
 type OpsgenieConfig struct {
-	APIKey string `json:"apiKey"`
+	APIKey                string `json:"apiKey"`
+	EuropeanServiceRegion bool   `json:"europeanServiceRegion"`
```

Comment:
If we decide to go with what I recommended, this should change to

```suggestion
	Region string   `json:"region" validate:"oneof=Europe US"`
```